pkg/trace/api/internal/header/headers.go

```diff
@@ -15,6 +15,11 @@ const (
 	// container where the request originated.
 	ContainerID = "Datadog-Container-ID"
 
+	// EntityID specifies the name of the header which contains entityID of the
+	// sender of the payload. This entityID can either be "cid-<container-id>", "pid-<pod.uid>", or
+	// "in-<cgroupv2-inode>" and is used to retrieve the container-id/pod-uid.
```

Comment:
Pod UID doesn't seem to be supported, can we avoid mentioning it here? Or at least document it will be supported in future versions.